### PR TITLE
[NETBEANS-3535] Corrected compiler warnings in XML Tools API UI project

### PR DESCRIPTION
**Pull request by mklaehn**
Fixed all compiler warnings concerning linter warnings types of

* rawtypes,
* static and
* unchecked.

### CODE REVIEW

File: ide/api.xml.ui/src/org/netbeans/spi/xml/cookies/SharedXMLSupport.java [hunk 1]
```diff
@@ -455,7 +455,7 @@ private void sendMessage(String message) {
         if (res==null) return null;
         NsHandler nsHandler = getNamespaces(is); 
         String[] namespaces = nsHandler.getNamespaces();
-        List loc = new ArrayList();
+        List<String> loc = new ArrayList();
```
Comment (r3ddr4g0n) [inline ide/api.xml.ui/src/org/netbeans/spi/xml/cookies/SharedXMLSupport.java]: List<String> loc = new ArrayList<>();

Comment (r3ddr4g0n) [inline ide/api.xml.ui/src/org/netbeans/spi/xml/cookies/SharedXMLSupport.java]: `List<String> loc = new ArrayList<>();`

Comment (mklaehn) [reply ide/api.xml.ui/src/org/netbeans/spi/xml/cookies/SharedXMLSupport.java]: Fixed. Good catch

**Comment by geertjanw** [comment]
Great, thanks for the clean ups.

**Comment by mklaehn** [comment]
> Great, thanks for the clean ups.

Thanks. Can I coount you as a reviewer?

**Comment by geertjanw** [comment]
This one looks good to me, approved.

**Comment by mklaehn** [comment]
Great.
Merging

**Pull request merged**